src/request/components/RequestDetailPanelRequest.tsx

```diff
@@ -39,18 +44,30 @@ export class Request extends React.Component<IRequestProps, {}> {
         return content;
     }
 
-    private renderRequestResponse(title: string, body: string, headers: { [key: string]: string }) {
+    private renderRequestResponse(title: string, body: string, headers: { [key: string]: string }, query?: { [key: string]: string }, formData?: { [key: string]: string }) {
+        const panels = [
+            { header: 'Headers', renderContent: () => this.renderHeaders(headers) },
+            { header: 'Body', renderContent: () => this.renderBody(body) }
+        ];
+
+        if (!_.isEmpty(query) || !_.isEmpty(formData)) {
+            panels.push({ header: 'Params', renderContent: () => this.renderParams(query, formData) });
+        }
+        
         return (
             <div className='tab-request-response-panel'>
                 <div className='tab-request-response-title'>{title}</div>
                 <br />
                 <TabbedPanel>
-                    <TabPanel header='Headers'>
-                        { this.renderHeaders(headers) }
-                    </TabPanel>
-                    <TabPanel header='Body'>
-                        { this.renderBody(body) }
-                    </TabPanel>
+                    { 
+                        panels.map(panel => {
+                            return (
+                                <TabPanel header={panel.header}>
+                                    { panel.renderContent() }
+                                </TabPanel>
+                            );
+                        }) 
+                    }
```

Comment:
I'm assuming you know this but I'm guessing there are going to be some merge conflicts here with the other PRs you have out.